content/docs/command-reference/plots/index.md

```diff
@@ -64,8 +64,8 @@ epoch, AUC, loss
 37, 0.92302, 0.0299015
 ```
 
-In hierarchical file formats (JSON or YAML), an array of consistent objects is
-expected: every object should have the same structure.
+Hierarchical file formats such as JSON and YAML consists of an array of objects
+where each object should contain all fields that are required for the plot.
```

Comment:
OK it's good that this is short again but this text doesn't seem to explain what we're trying to cover here (how to put plots data in hierarchical formats). In fact it sounds like we're suddenly describing what JSON and YAML files are in general.

For example, "contain all fields that are required for the plot" is implied, what we need to explain is how.

Reply:
Things to summarize (from https://github.com/iterative/dvc.org/pull/1961#pullrequestreview-540688242):

- json/yaml files shouodl contain an array of objects
- every object should have the same base structure
- X and Y fields should be part of that base structure
- extra elements will be ignored (not an error though)

Reply:
Finishing this up in https://github.com/iterative/dvc.org/pull/1961#pullrequestreview-555140849 below.